Src/Newtonsoft.Json/JsonWriter.cs

```diff
@@ -332,7 +332,7 @@ public CultureInfo Culture
         }
 
         /// <summary>
-        /// Creates an instance of the <c>JsonWriter</c> class. 
+        /// Initializes a new instance of a <c>JsonWriter</c>-derived class.
```

Comment:
Hmm, is this how protected base constructors are generally documented on MSDN? I'd like to follow MS's lead in commenting style.